src/views/NewReview.vue

```diff
@@ -6,23 +6,29 @@
     </v-app-bar>
 
     <br />
-    <v-avatar size="100">
+    <br />
+    <v-avatar size="90">
       <v-img id="class-image" :src="clas.pictureSources[0]" />
     </v-avatar>
+    <hr style="height:15pt; visibility:hidden;" />
+    <h2 style="color:#403D3D;">{{ clas.name }}</h2>
+
+    <p style="font-size:75%; color:#625B5A;">How do you like the class?</p>
 
     <v-rating
       v-model="ratings"
       background-color="orange lighten-3"
       color="orange"
-      medium
+      size="30"
```

Comment:
Try to not used fixed sizes, as this will not scale as well on different mobile dimensions than "medium"